hw/ip/csrng/data/csrng.hjson

```diff
@@ -207,6 +207,43 @@
         }
       ]
     },
+    { name: "INT_STATE_NUM",
+      desc: "Internal state number register",
+      swaccess: "rw",
+      hwaccess: "hro",
+      regwen: "REGEN",
+      fields: [
+        {
+            bits: "3:0",
+            name: "INT_STATE_NUM",
+            desc: '''
+                  Setting this field will set the number for which internal state can be
+                  selected for a read access. In normal operation, this register is hidden from
+                  firmware access. Only in life cycle mode set to "hw_debug" can the value
```

Comment:
About the "hidden in normal operation" note: Do you really mean that, or do you mean that `INT_STATE_VAL` is hidden when not in debug mode?

Reply:
@rswarbrick thanks for reviewing this! Rewrote to be clearer.